tests/scale-tests/results/Concurrent_Drain_Results.md

```diff
@@ -0,0 +1,302 @@
+# Concurrent Drain Operations Scale Test Results
+
+**Cluster:** 1503 nodes (3 system + 1500 customer aws-cpu-m7i.xlarge nodes)  
+**NVSentinel Version:** v0.4.0  
+**Node Drainer Config:** `evictionTimeoutInSeconds=60`, `mode=Immediate` for test-workload namespace  
```

Comment:
Can we see if `mode=AllowCompletion` also runs into the client side rate limits?

Reply:
make sense? Or did I misunderstand your question/something seems odd?